src/routes/controllers/articles.js

```diff
@@ -0,0 +1,106 @@
+import crypto from 'crypto';
+import slug from 'slug';
+import models from '../../db/models';
```

Comment:
There should be space between installed packages and local modules.

Reply:
resolved.